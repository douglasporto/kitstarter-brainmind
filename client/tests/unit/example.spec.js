import { shallowMount } from '@vue/test-utils';
import bmApp from '../../src/App';

describe('App.vue', () => {
  it('renders App', () => {
    // arrange
    //const wrapper = shallowMount(App); simple
    const wrapper = shallowMount(bmApp, {
      stubs: ['router-link', 'router-view']
    });

    // assert
    expect(wrapper.html()).toMatchSnapshot();
  });
});
