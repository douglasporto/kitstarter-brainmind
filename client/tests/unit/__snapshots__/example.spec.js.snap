// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue renders App 1`] = `
<div id="app">
  <router-view-stub></router-view-stub>
</div>
`;
